package.json

```diff
@@ -10,9 +10,10 @@
     "mastarm": "bin/mastarm"
   },
   "scripts": {
-    "cover": "bin/mastarm test --env test --test-environment node --coverage --coverage-paths bin",
+    "cover": "yarn run test -- --coverage --coverage-paths bin",
+    "lint": "bin/mastarm lint",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "bin/mastarm lint && bin/mastarm test --env test --test-environment node"
+    "test": "bin/mastarm test --env test --test-environment node"
```

Comment:
ok, fixed that.  One of the reasons I did this was that linting wasn't being ran on travis.  Typically we run the coverage and not the testing, so I made a separate command to run linting.